### ISSUE TITLE
AC Source current sense is determined without regard for the "phase shift" controls

### ISSUE THREAD

**Issue by samreid** [phetsims/circuit-construction-kit-common#805]
From https://github.com/phetsims/circuit-construction-kit-common/issues/800#issuecomment-991212081, I said:

In testing a pair of AC voltage circuits, I observed the following:

### Connecting at arbitrary time, they are in phase as desired:
![image](https://user-images.githubusercontent.com/679486/145625697-5e2016e3-14ba-4d60-8a01-40bbe8b5b6c4.png)

### Then, changing the phase puts them out of phase, as expected:
![image](https://user-images.githubusercontent.com/679486/145625811-a42b59e6-a0bc-4871-aaa7-66cb816eb500.png)

### However, leaving the "phase shift" controls out of phase, then open and closing a switch puts them back in phase, because the sense determining algorithm doesn't look at the "phase shift" controls.  Hopefully that is OK
![image](https://user-images.githubusercontent.com/679486/145625987-472ebf39-523a-4c5c-bce1-6fc4c716aab8.png)

**Comment by samreid** [comment]
@kathy-phet said:

> @samreid - Can you make a new issue for this? So we can fix it when we do PhET-iO add. I think its fixable.

**Comment by samreid** [comment]
I’ll open an issue, but not sure how we will handle the case when they are 90 degrees out of phase.

**Comment by arouinfar** [comment]
I am unable to reproduce this in master. I created two identical circuits (see below) and introduced a phase shift in the bottom circuit. I tried several different combinations of opening/closing the two switches, and no matter what I did, the bottom circuit maintained the correct phase shift (90°). I repeated for other phase shift values, including introducing a shift in both sources, and the circuits maintained the correct relative phase.

![image](https://user-images.githubusercontent.com/8419308/223281994-3a949d24-532c-4ac7-a0e6-efa049de96a7.png)
